carriers.json

```diff
@@ -1,30 +1,20 @@
 {
-  "Accessyou": "mgw.mmsc1.hkcsl.com",
```

Comment:
I made this change a while ago. I'll merge the current master branch with my fork and resubmit my PR